content/blog/10-things-you-can-do-with-neo/index.md

```diff
@@ -0,0 +1,192 @@
+---
+title: "10 Things You Can Do With Pulumi Neo"
```

Comment:
```suggestion
title: "10 Things You Can Do With Our Infrastructure Agent, Neo"
```

^^^ better for folks who don't yet know what Neo is?

Reply:
makes sense! I'll leave the meta image though if that is good with you